rts/motoko-rts/src/types.rs

```diff
@@ -444,20 +438,19 @@ pub const TAG_SOME: Tag = 13;
 pub const TAG_VARIANT: Tag = 15;
 pub const TAG_BLOB: Tag = 17;
 pub const TAG_FWD_PTR: Tag = 19; // Only used by the copying GC - not to be confused with forwarding pointer in the header used for incremental GC.
-pub const TAG_BITS32: Tag = 21;
-pub const TAG_BIGINT: Tag = 23;
-pub const TAG_CONCAT: Tag = 25;
-pub const TAG_REGION: Tag = 27;
-pub const TAG_ONE_WORD_FILLER: Tag = 29;
-pub const TAG_FREE_SPACE: Tag = 31;
+pub const TAG_BIGINT: Tag = 21;
+pub const TAG_CONCAT: Tag = 23;
+pub const TAG_REGION: Tag = 25;
+pub const TAG_ONE_WORD_FILLER: Tag = 27;
+pub const TAG_FREE_SPACE: Tag = 29;
 
 // Special value to visit only a range of array fields.
 // This and all values above it are reserved and mean
 // a slice of an array object (i.e. start index) for
 // purposes of `visit_pointer_fields`.
 // Invariant: the value of this (pseudo-)tag must be
 //            higher than all other tags defined above
-pub const TAG_ARRAY_SLICE_MIN: Tag = 32;
+pub const TAG_ARRAY_SLICE_MIN: Tag = 30;
```

Comment:
Curious - why is it ok for this Tag to be even, when all others must be odd? Ah, they only need to be odd for the marc-compact GC?

Reply:
Yes, this is only historical for the mark-compact GC. And even with the mark-compact GC, we use an even slice minimum value as this is only a lower boundary to distinguish slice information from the actual tag values.

Reply:
(maybe add that explanation in a comment)